src/internal.c

```diff
@@ -5223,7 +5223,7 @@ int RsaDec(WOLFSSL* ssl, byte* in, word32 inSz, byte** out, word32* outSz,
 int RsaEnc(WOLFSSL* ssl, const byte* in, word32 inSz, byte* out, word32* outSz,
     RsaKey* key, buffer* keyBufInfo)
 {
-    int ret = BAD_FUNC_ARG;
+    int ret = WC_NO_ERR_TRACE(BAD_FUNC_ARG);
```

Comment:
That doesn't look right.
Idea falls done some when the error is preset and set to 0 later.

Reply:
That's actually fine there.  `ret` will be set to an explicit+meaningful code by the called routines, mainly `wc_RsaPublicEncrypt()`.

Also bear in mind we're not aiming for perfection here...